### PR TITLE
OSDOCS-621 - Adding 'Copy login command' note

### PR DESCRIPTION
**Pull request by pneedle-rh**
Applies to `main`, `enterprise-4.6`, `enterprise-4.7`, `enterprise-4.8` and `enterprise-4.9`.

This pull request relates to https://issues.redhat.com/browse/OSDOCS-621.

The preview is [here](https://deploy-preview-35385--osdocs.netlify.app/openshift-enterprise/latest/cli_reference/openshift_cli/getting-started-cli.html#cli-logging-in_cli-developer-commands).

### CODE REVIEW

File: modules/cli-logging-in.adoc [hunk 1]
```diff
@@ -52,4 +52,9 @@ Welcome! See 'oc help' to get started.
 <2> Enter whether to use insecure connections.
 <3> Enter the user's password.
 
+[NOTE]
+====
+If you are logged in to the {product-title} web console, you can generate an `oc login` command that includes your token and server information. You can use the command to log in to the {product-title} CLI without the interactive prompts. To generate the command, select *Copy login command* from the drop-down menu at the top right of the web console.
```
Comment (bobfuru) [inline modules/cli-logging-in.adoc:57]: I _think_ we generally refer to it simply as "the web console".

```suggestion
If you are logged in to the web console, you can generate an `oc login` command that includes your token and server information. You can use the command to log in to the {product-title} CLI without the interactive prompts. To generate the command, select *Copy login command* from the drop-down menu at the top right of the web console.
```

**Review by bobfuru — COMMENTED** [review]
One minor nit but otherwise this LGTM!

File: modules/cli-logging-in.adoc (hunk 1 above)
Comment (pneedle-rh) [reply modules/cli-logging-in.adoc:57]: @bobfuru thank you for your review and for highlighting this. I have now resolved this suggestion.

**Comment by pneedle-rh** [comment]
@yapei can you please review this PR from a QE perspective? Thanks!

**Pull request merged**